boards/arm/96b_carbon_nrf51/Kconfig.defconfig

```diff
@@ -9,4 +9,11 @@ if BOARD_96B_CARBON_NRF51
 config BOARD
 	default "96b_carbon_nrf51"
 
+if BT
+
+config BT_CTLR
+	default y
+
+endif # BT
```

Comment:
Could simplify these to make things a bit less spammy:

```
config BT_CTLR
	default BT
```

Reply:
That's exactly the sort of cleaner solution I was hoping would get discussed around https://github.com/zephyrproject-rtos/zephyr/pull/12769#pullrequestreview-196824143.  A fourth way to do it....

Reply:
Ah, nice suggestion! let me do that!